app/components/administrative-unit-status.js

```diff
@@ -0,0 +1,13 @@
+import Component from '@glimmer/component';
+
+export default class AdministrativeUnitStatusComponent extends Component {
```

Comment:
Optional: I know I pushed in this direction, but it seems the status is related to the organization, not "administrative unit". So renaming the component to `<OrganizationStatus>` might make sense?